### ISSUE TITLE
Feature Request - Add Save and Close button on appropriate editable pop-ups

### ISSUE THREAD

**Issue by mkoo** [ArctosDB/arctos#6988]
A minor improvement but please add a second **Save and Close** button feature on all appropriate edit record sections?
![Firefox_Screenshot_2023-11-18T20-35-49 728Z](https://github.com/ArctosDB/arctos/assets/2523089/675081e3-1ad4-4ef2-8dc4-7eb943329bca)

Often I cannot tell if my Save click did anything and I suspect others are in the same boat. So they click on it like a banshee causing all sorts of bad traffic. If we have a Save and Close then the pop-up window would go away and reduce the amount of unnecessary clicks!

Proposed windows for the Save and Close button:
- Agents
- Parts
- Attributes
- Identifiers (aka Other Ids)
(The other editable pop-up windows have their own editing forms that have actions not requiring this kind of user feedback)

We should keep the close X in the far right corner but it's still a long drag for our mouse after furiously clicking Save!
![Firefox_Screenshot_2023-11-18T22-28-01 989Z](https://github.com/ArctosDB/arctos/assets/2523089/9c5a1ff5-df00-41aa-b50c-22cf0edf48ff)

**Comment by ewommack** [comment]
Yes please! I think this confuses my students all the time.

**Comment by dustymc** [comment]
merge-->https://github.com/ArctosDB/arctos/issues/6684

I'm merging this as an opportunity to discuss bigger changes, and perhaps to catch a potential problem before it comes to exist.
